packages/contentstack-export/src/config/index.ts

```diff
@@ -25,6 +25,7 @@ const config: DefaultConfig = {
       'environments',
       'extensions',
       'webhooks',
+      'taxonomies',
```

Comment:
@aman19K I hope this is considering the dependency of the plugin hierarchy

Reply:
Yes @netrajpatel